_posts/fr/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,409 @@
+  devraient être activés.
+
+## Bitcoin Core PR Review Club
+
+*Dans cette section mensuelle, nous résumons une récente réunion du [Bitcoin Core PR Review Club][],
+en mettant en lumière certaines des questions et réponses importantes. Cliquez sur une question
+ci-dessous pour voir un résumé de la réponse de la réunion.*
+
+[Add PayToAnchor(P2A), OP_1 <0x4e73>, comme script de sortie standard pour dépenser][review club
+30352] est une PR de [instagibbs][gh instagibbs] qui introduit un nouveau type de script de sortie
+`TxoutType::ANCHOR`. Les sorties d'ancrage ont un script de sortie `OP_1 <0x4e73>` (résultant en une
+adresse [`bc1pfeessrawgf`][mempool bc1pfeessrawgf]). Rendre ces sorties standard facilite la
+création et la transmission de transactions qui dépensent à partir d'une sortie d'ancrage.
+
+{% include functions/details-list.md
+  q0="Avant que `TxoutType::ANCHOR` soit défini dans cette PR, quel `TxoutType` serait classifié comme
+  `scriptPubKey` `OP_1 <0x4e73>` ?"
+  a0="Puisqu'il consiste en un opcode de poussée de 1 octet (`OP_1`) et un 2 octet
+  poussée de données (`0x4e73`), il s'agit d'une sortie témoin v1 valide. Comme elle ne fait pas 32
+  octets, elle ne se qualifie pas comme `WITNESS_V1_TAPROOT`, se classant donc par défaut comme
+  `TxoutType::WITNESS_UNKNOWN`."
+  a0link="https://bitcoincore.reviews/30352#l-18"
+
+  q1="En se basant sur la réponse à la question précédente, serait-il standard de créer ce type de
+  sortie ? Qu'en est-il pour la dépenser ? (Indice : comment [`IsStandard`][gh isstandard] et
+  [`AreInputsStandard`][gh areinputsstandard] traitent-ils ce type ?)"
+  a1="Puisque `IsStandard` (qui est utilisé pour vérifier les sorties) ne considère que
+  `TxoutType::NONSTANDARD` comme non standard, le créer serait standard. Puisque `AreInputsStandard`
+  considère qu'une transaction qui dépense depuis un `TxoutType::WITNESS_UNKNOWN` est non standard, il
+  ne serait pas standard de le dépenser."
+  a1link="https://bitcoincore.reviews/30352#l-24"
+
+  q2="Avant cette PR, avec les paramètres par défaut, quels types de sorties peuvent être _créés_ dans
+  une transaction standard ? Est-ce la même chose que les types de scripts qui peuvent être _dépensés_
+  dans une transaction standard ?"
+  a2="Tous les `TxoutType` définis à l'exception de `TxoutType::NONSTANDARD` peuvent être créés. Tous
+  les `TxoutType` définis à l'exception de `TxoutType::NONSTANDARD` et `TxoutType::WITNESS_UNKNOWN`
+  sont autorisés à être dépensés (bien qu'il soit impossible de dépenser `TxoutType::NULL_DATA`)."
+  a2link="https://bitcoincore.reviews/30352#l-42"
+
+  q3="Définissez _sortie ancrée_, sans mentionner les transactions du réseau Lightning (essayez d'être
+  plus général)."
+  a3="Une sortie ancrée est une sortie supplémentaire créée sur des transactions pré-signées pour
+  permettre l'ajout de frais via CPFP au moment de la diffusion. Voir aussi [topic anchor outputs]
```

Comment:
I believe this topic anchor output link should be attached to some text

Reply:
@Copinmalin

Reply:
Hello, 
Sorry I don't get your point? What should be done?

Reply:
Hello,

Sorry I don't get your point. What should be modified?

Reply:
@bitschmidty OK but which one ?

Reply:
I think we would want something like [<french text for "anchor outputs">][topic anchor outputs]

Reply:
@bitschmidty : done

Can I purpose a PR for english version ?

Reply:
That would be fine, yes

Reply:
Done : 
https://github.com/bitcoinops/bitcoinops.github.io/pull/1893